go/adbc/driver/snowflake/snowflake_database.go

```diff
@@ -30,6 +32,7 @@ import (
 	"github.com/apache/arrow-adbc/go/adbc"
 	"github.com/apache/arrow-adbc/go/adbc/driver/driverbase"
 	"github.com/snowflakedb/gosnowflake"
+	"github.com/youmark/pkcs8"
```

Comment:
We need to regenerate the license for this, I believe

Reply:
yes, because Go doesn't have support for encrypted PKCS8 keys